lib/tasks/licenses.rake

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+namespace :licenses do
+
+  report_path = Rails.root.join('doc/licenses/licenses.xml')
+
+  desc 'Generates a report with the dependencies and their licenses'
+  task :report do
+    $stdout.puts 'Generating report...'
+    exec license_finder_report
```

Comment:
btw why use `exec` instead of `system` here?
```suggestion
    system license_finder_report, exception: true
```

While as a last task, `exec` makes sense because we don't run anything after, if we have multiple rake tasks on the command line, this will terminate execution of any further list tasks.

Maybe unlikely but I think it is proper way to do.

P.S. `exception: true` is available from ruby 2.6 but we all use 2.6+ and we don't run this task on legacy.

Reply:
Short question: it was like this before 😄 I'll change it.